CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### AppCenter
 
+* **[Feature]** Allow users to set userId for crashes and error logs.
```

Comment:
@elamalani should we mention there is no backend support yet for App Center?

Reply:
I'll update it